qunetsim/components/network.py

```diff
@@ -584,17 +590,30 @@ def _process_queue(self):
                 Logger.get_instance().error(
                     "route couldn't be calculated, value error")
             except Exception as e:
+                import traceback
+                traceback.format_exc()
                 Logger.get_instance().error('Error in network: ' + str(e))
 
+        while True:
```

Comment:
In the previous version, it was possible to terminate the network by sending a stop signal. It seems this was removed. Is it still possible to stop the network in the same way?